src/components/resources/Resource.js

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+
+export const Resource = ({currentRes}) => {
+  return (
+    <StyledRescourceSection>
+      <StyledRescourceIcon
+        alt={currentRes.name}
+        src={'../images/icons/icon_' + currentRes.name.charAt(0).toLowerCase() + currentRes.name.slice(1) + '.jpg'}
```

Comment:
I think the part `currentRes.name.charAt(0).toLowerCase() + currentRes.name.slice(1)` is quite confusing at this place.

You can think about two different ways:
1. You could either add another key-value-pair to the const `mainResources` in App.js, something like `lowercaseName: 'money' and use this key here or
2. Compute the value in line 4 above the return, something like this:

```jsx
const lowercaseName = currentRes.name.charAt(0).toLowerCase() + currentRes.name.slice(1);
... src={'../images/icons/icon_' + lowercaseName + '.jpg'}
```

I would choose option 1 if there are several places where I need the lowercase name value, option 2 if it is only necessary in this Resource component.

Reply:
thanks. done